packages/IMAPClient-Core.package/ICEndPoint.class/instance/connectAndLoginWith..st

```diff
@@ -6,9 +6,9 @@ connectAndLoginWith: anICAccountInfo
 	[self
 		connect;
 		login]
-	valueWithin: 2 seconds 
+	valueWithin: 5 seconds 
```

Comment:
What exactly does this change?

Reply:
OWA works

Reply:
Please provide some more details...

Reply:
OWA is notoriously slow and for some users this provided problems, preventing login

Reply:
So why did it work for Julia and me then?